### DISCUSSION TITLE
Not redirecting

### DISCUSSION THREAD

**Discussion by PeterJChaaya** [community/community#119007] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://github.com/community/community/assets/166909870/ba887ba2-fb13-4575-98a7-b7ecd866a985

Its not redirecting, how can i fix that?

**Comment by cedr0u** [comment]
if the redirection bugs, copy and paste the link yourself in a new tab, or else check that the link is still functional.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @PeterJChaaya ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
